public/index.html

```diff
@@ -9,7 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/Collaboratio.svg" />
```

Comment:
The collaboratio logo seems to have a full white background, could you check that ?